src/devices/controller/controller.cpp

```diff
@@ -1,66 +1,115 @@
 #include <FastLED.h>
 
 #include "Arduino.h"
+#include "FakeLedManager.hpp"
+#include "analog-button.h"
+#include "arduino-timer.h"
+#include "arduino/RadioHeadRadio.hpp"
+#include "generic/NetworkManager.hpp"
+#include "generic/RadioStateMachine.hpp"
 
 // Pin definitions
 constexpr int kSwitchLeft = PA0;
```

Comment:
"switch" feels like a weird term here while we're at it.

Reply:
Agreed. The v1.1 board uses direct connections for each button, instead of this resistor divider. I have renamed this in the update for v1.1 hardware.